code/game/objects/items/devices/radio/radio.dm

```diff
@@ -66,6 +66,9 @@
 		var/turf/T = get_turf(src)
 		if(T && is_ground_level(T.z))
 			frequency = COLONY_FREQ
+			var/mob/living/carbon/human/human_spawning_on = locate(/mob/living/carbon/human) in T
+			if(human_spawning_on && human_spawning_on.first_xeno)
+				listening = 0
```

Comment:
Mega Spagget alert

Have it turn off in /datum/job/antag/xenos/proc/transform_to_xeno or something, or bust the radio key, i don't know, but radio init scanning/checking for unrelated stuff is kinda no-no...

Reply:
geev fix